cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/AbstractTicketRegistry.java

```diff
@@ -38,6 +42,26 @@
     @Qualifier("ticketCipherExecutor")
     private CipherExecutor<byte[], byte[]> cipherExecutor;
 
+    @Nullable
+    @Autowired(required = false)
+    @Qualifier("ticketDelegators")
+    private List<Pair<Class<? extends Ticket>, Constructor<? extends AbstractTicketDelegator>>> ticketDelegators = new ArrayList<>();
```

Comment:
this list is currently both `Autowired` by spring and manually initialized in the constructor. I also remember seeing you auto-register the OAuth `TicketDelegator`s when those are loaded by directly adding to the `ticketDelegators` array. The `Autowired` functionality feels inconsistent with the other 2 ways we add `TicketDelegator`s.

How do you envision this list being extended by other components that define their own custom `Ticket` classes?